### DISCUSSION TITLE
Finance: flask run - not running

### DISCUSSION THREAD

**Discussion by RobMDoe** [community/community#116864] [Programming Help]
### Select Topic Area

Flask run not running

### Body

All fine for many weeks. flask run opening in browser and / buy almost complete. But now flask run won't run.  Gets as far as:

* Serving Flask app 'app.py'
 * Debug mode: off
INFO: WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on https://robmdoe-symmetrical-broccoli-69964rrq55rfx5xp-5000.app.github.dev
INFO: Press CTRL+C to quit
INFO:  * Restarting with stat

Nothing at URL. CS50.ai suggestions followed. Can you help?

**Comment by Tarandeep9988** [comment]
same problem

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @RobMDoe!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by RobMDoe** [reply]
Thank you.
